app/factories/post_factory.rb

```diff
@@ -27,10 +28,11 @@ def parse_tags(header)
       result.join(', ')
     end
 
-    def create_post_model(post_contents, file_path)
+    def create_post_model(post_contents, file_path, ref)
       result = Post.new
 
       result.file_path = file_path
+      result.github_ref = ref
 
       # What this regular expression does is it matches two groups
       # The first group represents the header of the post which appears
```

Comment:
Here you specify two capture groups in a regex, and explain what the first group is for.  Why have the second if we don't use it?

Reply:
I tried removing the second group by making the regex `/---(.*)---\r\n|\r|\n(.*)/m` and there were matching problems without the second group on the newline characters. If we want to modify this I'd say we make a tech debt issue since that regex wasn't touched in this PR.

Reply:
Ah, I see. I think this is too small to slide into tech debt, so I'd just say make an indication of why the 2nd group exists in a comment.

Reply:
Alright, I updated the comment